test/integration/joint_controller_system.cc

```diff
@@ -144,6 +144,39 @@ TEST_F(JointControllerTestFixture,
   }
 }
 
+/////////////////////////////////////////////////
+// Tests that the JointController accepts joint velocity commands
+// See https://github.com/gazebosim/gz-sim/issues/1175
+TEST_F(JointControllerTestFixture,
+       GZ_UTILS_TEST_DISABLED_ON_WIN32(
+        JointVelocityMultipleJointsSubTopicCommand))
+{
+  using namespace std::chrono_literals;
+
+  // Start server
+  ServerConfig serverConfig;
+  const auto sdfFile = std::string(PROJECT_SOURCE_PATH) +
+    "/test/worlds/joint_controller.sdf";
```

Comment:
`gz::common::joinPaths`

Reply:
Got it, will edit now.

Reply:
I edited it also in all the rest of the file from the previous method to the now preferred joinPaths method.